datascience/tables.py

```diff
@@ -5735,9 +5771,35 @@ def __getitem__(self, i):
             return self._row(c[i] for c in self._table._columns.values())
 
         def __len__(self):
+            """
+            Returns the no.of rows in the table.
+
+            >>>  t = td.Table().with_columns({
+             'letter': ['a', 'b', 'c', 'z'],
+             'count':  [  9,   3,   3,   1],
+             'points': [  1,   2,   2,  10],
+            })
+            >>> td.Table().Rows(t).__len__()
```

Comment:
We can jus use the Python `len` function to demonstrate this function's logic

Reply:
Noted.